docs/cpp-runnable.md

```diff
@@ -4,7 +4,18 @@ id: cpp-runnable
 ---
 
 The `fuse` CLI provides the `run` subcommand for generating executables that
-can be used to test Fuse code without going throught a synthesis workflow.
+can be used to test Fuse code without going through a synthesis workflow.
+
+## Limitation
+
+The backend is meant to be used for functional testing and as such has a few
+important limitations.
+
+1. Exact bitwidth integer computation is not supported. Sized integers get
+   compiled to C++ `int`.
```

Comment:
Could be worth making a separate issue about addressing this limitation, which shouldn't be too hard (using [ac_int](https://www.mentor.com/hls-lp/downloads/ac-datatypes), perhaps).

Reply:
Adding this to the low priority tracker. This is slightly non trivial because the Json library can't extract the type directly using `get_arg`, but I think we can just use a cast.